docs/markdown/Getting Help/the-pants-community/maintainers.md

```diff
@@ -4,41 +4,84 @@ slug: "maintainers"
 excerpt: "What Pants maintainers do and how to become one."
 hidden: false
 createdAt: "2020-05-16T22:36:48.659Z"
-updatedAt: "2022-05-26T22:13:54.279Z"
+updatedAt: "2022-09-28T00:00:00.000Z"
 ---
-The Pants community has several Maintainers: people with a proven track record of contributions to the community and an ongoing commitment to the project, who guide the contributions of the wider community.
+Pants community Maintainers are contributors who have been recognized by the project as stewards of Pants. They have a proven record of contributions, are committed to ensuring the continued success of our project, and through their actions help to grow our Community.
 
-Maintainer responsibilities
----------------------------
+What is a maintainer?
+---------------------
 
-It is the responsibility of a maintainer to uphold the health and quality of the project.
+[Maintainers are the core group of trusted people with a long-term interest in Pants, who have made regular contributions for some time, and plan to continue to do so](doc:the-pants-community#maintainers). Maintainers are entrusted to make decisions on the future and direction of Pants. In doing so, it is their responsibility to uphold the health, culture, and quality of the project.
 
-- Maintainers are responsible for the quality of the contributions they approve.
-- Maintainers should raise objections to changes that may impact the performance, security, or maintainability of the project.
-- Maintainers should help shepherd changes through our contribution process.
-- Maintainers should maintain a courteous and professional demeanor when participating in the community.
-- Maintainers should be regular participants on our public communications channels.
+We do not expect a minimum commitment of time from Maintainers, nor do we expect maintainers to contribute in specific ways, rather, we expect them to act in a way that makes Pants better whenever they interact with the project and our community.
```

Comment:
At least in Github's preview, the underline immediately under this paragraph renders it as Very Large. Not sure whether that will carry over to readme, but was that intended?